[{"exception": "error", "inner": [{"provider": {"package": "oracle.jdbc.driver", "class": "DatabaseError", "method": "newSQLException", "filename": "DatabaseError", "line": "112"}, "caller": {"package": "org.eclipse.datatools.connectivity.drivers.jdbc", "class": "JDBCConnection", "method": "createConnection", "filename": "JDBCConnection", "line": "213"}}], "outer_pre": [], "outer_suf": [{"provider": {"package": "oracle.jdbc.driver", "class": "DatabaseError", "method": "newSQLException", "filename": "DatabaseError", "line": "112"}, "caller": {"package": "org.eclipse.datatools.connectivity.drivers.jdbc", "class": "JDBCConnection", "method": "createConnection", "filename": "JDBCConnection", "line": "213"}}], "all": [{"provider": {"package": "oracle.jdbc.driver", "class": "DatabaseError", "method": "newSQLException", "filename": "DatabaseError", "line": "112"}, "caller": {"package": "org.eclipse.datatools.connectivity.drivers.jdbc", "class": "JDBCConnection", "method": "createConnection", "filename": "JDBCConnection", "line": "213"}}]}]